theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -5,28 +5,14 @@ set -xe
 echo "--Apt Hook Setup"
 
 pbuilder_hooksdir=/etc/pbuilder/${os}64/hooks
```

Comment:
this directory doesn't contain *anything* specific to the `os` anymore, so we can move everything to `/etc/pbuilder/hooks` later.